wordpress-mobile/WordPress-iOS
# Remove unused source tag

Fixes #n/a

WPError had functionality for passing a sourceTag to Support. With refactors, that functionality is no longer used, and is handled elsewhere. This removes WPError sourceTag functionality.

To test:
The only functional change is getting the source origin for Zendesk. To test that:
- In `ZendeskUtils:getTags`, log out `sourceTagOrigin`.
- Run the app.
- Access Support from somewhere that has a `sourceTag`, like Create Site > Help.
- Select 'Contact Us'. 
- Verify the `sourceTag.origin` appears in the log.
